code/modules/xgm/xgm_gas_mixture.dm

```diff
@@ -19,10 +19,17 @@
 	///The last cached color of the gas mixture
 	var/tmp/cached_mix_color
 
-/datum/gas_mixture/New(_volume = CELL_VOLUME, _temperature = 0, _group_multiplier = 1)
-	volume = _volume
-	temperature = _temperature
-	group_multiplier = _group_multiplier
+/datum/gas_mixture/New(_volume, _temperature, _group_multiplier)
```

Comment:
This is so you can define a gas mixture at compile time. Also, using the args as default values was redundant because the values defined in the datum are already set to those.

Reply:
The New() override with those parameters was already there though, I just modified it? 
Also, I added a condition so that, update_values is only called if the gas list has any entries at all. Which should take care of most of any potential extra overhead you're afraid of.

Also, one advantage to consider is that if we can define gas_mixture subtypes with gases, volume, temp set at compilation we save on needing more runtime prep and initial gas lists instances.